camshare/camshare/PhotoAlbumViewController.swift

```diff
@@ -8,28 +8,34 @@
 
 import UIKit
 
+var selectedImageIndex: Int = 0
```

Comment:
why is this variable outside of the viewController?

Reply:
I made it global so that it can be accessed from the other viewController; This was to test if the album selected image would show on the next viewController

Reply:
Noooooo =( Dont do that you should be passing that data to the view controller when it gets created and not have a random variable just floating around. Look at other ways to do this.